app/src/main/res/layout-land/profile_chooser_fragment.xml

```diff
@@ -1,121 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<layout xmlns:android="http://schemas.android.com/apk/res/android"
-  xmlns:app="http://schemas.android.com/apk/res-auto">
-
-  <data>
-
-    <variable
-      name="viewModel"
-      type="org.oppia.android.app.profile.ProfileChooserViewModel" />
-  </data>
-
-  <androidx.constraintlayout.widget.ConstraintLayout
-    android:layout_width="match_parent"
-    android:layout_height="match_parent">
-
-    <androidx.core.widget.NestedScrollView
-      android:layout_width="match_parent"
-      android:layout_height="match_parent"
-      android:background="@color/profile_chooser_background"
-      android:overScrollMode="never"
-      android:scrollbars="none">
-
-      <androidx.constraintlayout.widget.ConstraintLayout
-        android:id="@+id/profile_chooser_container"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent">
-
-        <ImageView
-          android:id="@+id/profile_chooser_language_icon"
-          android:layout_width="48dp"
-          android:layout_height="48dp"
-          android:layout_marginEnd="36dp"
-          android:paddingStart="4dp"
-          android:paddingTop="20dp"
-          android:paddingEnd="20dp"
-          app:srcCompat="@drawable/ic_language_icon_grey_24dp"
-          android:visibility="gone"
-          app:layout_constraintEnd_toEndOf="parent"
-          app:layout_constraintTop_toTopOf="parent" />
-
-        <TextView
-          style="@style/Subtitle1ViewCenter"
-          android:minHeight="48dp"
-          android:paddingTop="20dp"
-          android:text="@string/profile_chooser_language"
-          android:textColor="@color/profile_chooser_grey_text"
-          android:visibility="gone"
-          app:layout_constraintEnd_toStartOf="@+id/profile_chooser_language_icon"
-          app:layout_constraintTop_toTopOf="parent" />
-
-        <TextView
-          android:id="@+id/profile_select_text"
-          style="@style/Heading1ViewStart"
-          android:layout_marginStart="36dp"
```

Comment:
The style attribute here differs from the one in portrait xml (mobile). you forgot to resolve this.